spec/models/organization_spec.rb

```diff
@@ -0,0 +1,42 @@
+require 'spec_helper'
+
+describe Organization do
+	it "1:  without http & https" do
+		org = Organization.new
+		org.web = "www.casa.com"
+		org.ensure_url
+		expect(org.web).to eq "http://www.casa.com"
+	end
+	it "2: with http" do
+		org = Organization.new
+		org.web = "http://www.casa.com"
+		org.ensure_url
+		expect(org.web).to eq "http://www.casa.com"
+	end
+		it "3: with https" do
+		org = Organization.new
+		org.web = "https://www.casa.com"
+		org.ensure_url
+		expect(org.web).to eq "https://www.casa.com"
+	end
```

Comment:
`end` at 21, 1 is not aligned with `it "3: with https" do` at 16, 2<br>Tab detected.